### PR TITLE
Allow adding permissions on Page & factorize code with adaccount

### PR DESCRIPTION
**Pull request by olasfar**

### CODE REVIEW

**Comment by m-vdb** [comment]
@olasfar we don't want to diverge too much from the original FB SDK. Have you opened a PR on their end? Have you talked to them to see if they were willing to accept PRs on this?

File: facebookads/adobjects/adaccount.py [hunk 1]
```diff
@@ -36,10 +36,22 @@
 
 class AdAccount(
     AdAccountMixin,
+    HasCreateUserPermission,
     AbstractCrudObject,
     HasAdLabels,
 ):
 
+    ROLE_ENUM = [
```
Comment (m-vdb) [inline facebookads/adobjects/adaccount.py]: `PERMISSION_ROLE_ENUM`

File: facebookads/mixins.py [hunk 2]
```diff
@@ -213,3 +217,37 @@ def __setitem__(self, key, value):
             )
         else:
             super(ValidatesFields, self).__setitem__(key, value)
+
+
+class HasCreateUserPermission(object):
```
Comment (m-vdb) [inline facebookads/mixins.py:222]: docstring

Comment (olasfar) [reply facebookads/mixins.py:222]: ✅

**Comment by olasfar** [comment]
https://github.com/facebook/facebook-python-ads-sdk/pull/258

**Comment by m-vdb** [comment]
@olasfar don't forget to create a git tag for this version, we use tags in [work4us' requirements.txt](https://github.com/Work4Labs/work4us/blob/master/requirements.txt#L38)

**Pull request merged**